tunnel.go

```diff
@@ -130,10 +66,21 @@ func newTunnel(dialer httpstream.Dialer, config TunnelConfig) (*Tunnel, error) {
 	case err = <-errChan:
 		return nil, microerror.Mask(err)
 	case <-pf.Ready:
-		return tunnel, nil
+		return t, nil
 	}
 }
 
+// Close disconnects a tunnel connection. It always returns nil error to fulfil
+// io.Closer interface.
+func (t *Tunnel) Close() error {
+	close(t.stopChan)
+	return nil
+}
+
+func (t *Tunnel) LocalAddress() string {
+	return fmt.Sprintf("127.0.0.1:%d", t.localPort)
+}
```

Comment:
I addressed a TODO in the `helmclient` project as well. So the tunnel exposes its own local address and nobody has to make this up on their own outside of their business domain.